pennylane/qaoa/cycle.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Methods for finding max weighted cycle of weighted directed graphs
+"""
+
+from typing import Dict, Tuple
+import networkx as nx
+
+
+def edges_to_wires(graph: nx.Graph) -> Dict[Tuple[int], int]:
```

Comment:
```suggestion
def edges_to_wires(graph: nx.Graph) -> Dict[Tuple, int]:
```
Now I think about it, they're not always ints right?

Reply:
Yes good point, we have applied this to cases where we have tuples of strings here.